Jellyfin.Server.Implementations/JellyfinDbProvider.cs

```diff
@@ -13,19 +15,27 @@ public class JellyfinDbProvider
     {
         private readonly IServiceProvider _serviceProvider;
         private readonly IApplicationPaths _appPaths;
+        private readonly ILogger<JellyfinDbProvider> _logger;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="JellyfinDbProvider"/> class.
         /// </summary>
         /// <param name="serviceProvider">The application's service provider.</param>
         /// <param name="appPaths">The application paths.</param>
-        public JellyfinDbProvider(IServiceProvider serviceProvider, IApplicationPaths appPaths)
+        /// <param name="logger">The logger.</param>
+        public JellyfinDbProvider(IServiceProvider serviceProvider, IApplicationPaths appPaths, ILogger<JellyfinDbProvider> logger)
         {
             _serviceProvider = serviceProvider;
             _appPaths = appPaths;
+            _logger = logger;
 
             using var jellyfinDb = CreateContext();
-            jellyfinDb.Database.Migrate();
+            if (jellyfinDb.Database.GetPendingMigrations().Any())
+            {
+                _logger.LogInformation("There are pending EFCore migrations in the database. Applying... (This may take a while, do not stop Jellyfin)");
```

Comment:
EF Core should log this itself as well, so we might just open up that SourceContext to the output?

Reply:
Added in [CoreAppHost](https://github.com/jellyfin/jellyfin/pull/6049/files#diff-4d188b93a5a2169482c7c60b65a4fd098d2131c2b8c48622abdfb80efe125138R77)